README.md

```diff
@@ -60,42 +60,37 @@ You may also wish to visit Intel Media Server Studio [support page](https://soft
 Media SDK test and sample applications may require additional software packages (for example, X Server, Wayland, LibDRM, etc.) to be functional.
 
 # How to build
-## Requirements
- - Git* (with [LFS](https://git-lfs.github.com/) support)
```

Comment:
Are you sure to delete requirements at all?
Especially in case of git **lfs**. I see that some people don`t install lfs and after that they wonder that the documentation is "not work".  :)

Reply:
I am absolutely sure to remove "requirements" section since it in any case don't list all the requirements. The only up to date list of the requirements is configure scripts. However, let's preserve note about Git with LFS. I will move it to the section where we describe how to get sources.